armory/data/utils.py

```diff
@@ -14,21 +14,31 @@
 from botocore import UNSIGNED
 from botocore.client import Config
 
+from armory.data.progress_percentage import ProgressPercentage
 
 logger = logging.getLogger(__name__)
 
 
-def download_file_from_s3(bucket_name: str, key: str, local_path: str):
+def download_file_from_s3(
+    bucket_name: str, key: str, local_path: str, progress_bar=False
```

Comment:
Would prefer if we remove API bloat and just always use progress_bar. Any reason not to?

Reply:
Agreed, this option seems unnecessary.